android/app/src/main/kotlin/coop/polypoly/polypod/features/Feature.kt

```diff
@@ -20,6 +20,10 @@ class Feature(
         get() = runCatching { Color.parseColor(manifest.primaryColor) }
             .getOrDefault(0)
 
+    val thumbnailColor: Int
```

Comment:
Here thumbnailColor is an Int...

Reply:
That's how it works on Android to the best of my knowledge. Plus we already use `Int` values for `primaryColor`, would prefer not to risk changing that while adding a fix during feature freeze :D